### DISCUSSION TITLE
finding unique files on second file system

### DISCUSSION THREAD

**Discussion by PaulSorenson** [qarmin/czkawka#708] [Ideas]
find files on an old backup which are unique.

Eg

- A is current workstation /home/user
- B is backup of an old workstation /home/user

I would like to find files on B that are unique across A and B so I can copy them somewhere safe before reusing the backup media.
If the gui logic does not support this - is the cache in a format that I could read with say python and find the target files myself?

**Comment by likeweiss** [comment]
I have the same question. Was anyone able to figure out if this is doable?

**Comment by loganr82** [comment]
Duplicate all of what you have and on it do this: Search for matches, use "select all" to highlight every match, delete all matches then you are left with whatever is unique so you know what to search out in your actual filesystems.
